eng/scripts/Language-Settings.ps1

```diff
@@ -6,7 +6,7 @@ $LanguageDisplayName = "go"
 function Get-GoModuleVersionInfo($modPath)
 {
   $NO_PREFIX_VERSION_LINE_REGEX = ".+\s*=\s*`"(?<bad_version>$([AzureEngSemanticVersion]::SEMVER_REGEX))`""
-  $VERSION_LINE_REGEX = ".+\s*=\s*`".*v(?<version>$([AzureEngSemanticVersion]::SEMVER_REGEX))`""
```

Comment:
This is just changing the named group in a RegEx which isn't used outside of this regex and matching. Thus this is pretty much a noop, what is it that you are trying to update?

Reply:
The variable name has changed from `version` to `moduleVersion`, I am looking for the regex that searches for this

Reply:
Looking at this more in depth, I don't think there is a check on the variable `version`. Is that correct @weshaggard ? It looks like it's doing a search for semantic versioning in a specific file.

Reply:
That is correct this regex is only looking for something that looks like "<anyname>=v<version>". If it finds that in one of the file names we look in hen it assumes that is the module version. If we trying to be more prescriptive then we would want to do something like `moduleVersion\s*=\s*`".*v(?<version>$([AzureEngSemanticVersion]::SEMVER_REGEX))`" `